<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="CheckOpiFormat-20171122143019" tests="9" time="0.000">
	<testcase classname="CheckOpiFormat" name="test_GIVEN_a_label_THEN_it_ends_in_a_colon" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_a_label_outside_a_grouping_container_THEN_it_is_title_case" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_a_label_within_a_grouping_container_THEN_it_is_sentence_case" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_an_opi_file_with_labels_WHEN_checking_the_background_colour_THEN_it_is_the_isis_background" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_an_opi_file_with_led_WHEN_checking_off_colour_THEN_it_is_the_isis_led_off_colour" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_an_opi_file_with_led_WHEN_checking_on_colour_THEN_it_is_the_isis_led_on_colour" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_an_opi_file_with_textbox_WHEN_checking_background_colour_THEN_it_is_the_isis_textbox_background" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_an_opi_file_with_textbox_WHEN_checking_foreground_colour_THEN_it_is_the_isis_textbox_foreground" time="0.000"/>
	<testcase classname="CheckOpiFormat" name="test_GIVEN_an_opi_file_with_grouping_containers_WHEN_checking_the_background_colour_THEN_it_is_the_isis_background" time="0.000">
		<error message="On line 554, text 'Sensor 5', colour was not correct.
On line 636, text 'Sensor 6', colour was not correct.
On line 718, text 'Sensor 7', colour was not correct." type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "check_opi_format.py", line 234, in test_GIVEN_an_opi_file_with_grouping_containers_WHEN_checking_the_background_colour_THEN_it_is_the_isis_background
    self._assert_colour_correct("background_color", "groupingContainer", ["ISIS_OPI_Background"])
  File "check_opi_format.py", line 231, in _assert_colour_correct
    self.fail("\n".join(["On line {}, text '{}', colour was not correct.".format(*error) for error in errors]))
AssertionError: On line 554, text 'Sensor 5', colour was not correct.
On line 636, text 'Sensor 6', colour was not correct.
On line 718, text 'Sensor 7', colour was not correct.
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
